extraction/src/main/scala/org/broadinstitute/monster/encode/extraction/Args.scala

```diff
@@ -20,5 +22,22 @@ case class Args(
   @HelpMessage(
     "Batch size that defines the number of elements in a batch when making certain API calls"
   )
-  batchSize: Long
+  batchSize: Long,
+  @HelpMessage(
+    "Initial query to target a specific entry-point to the pipeline."
+  )
+  initialQuery: List[(String, String)] = List("organism.name" -> "human")
 )
+
+object Args {
+
+  implicit val customArgParser: ArgParser[(String, String)] =
```

Comment:
Now that it's working, would be more specific with the name
```suggestion
  implicit val tupleParser: ArgParser[(String, String)] =
```